apis/components/v1/dashboard_types.go

```diff
@@ -43,6 +45,9 @@ type DashboardSpec struct {
 // DashboardStatus defines the observed state of Dashboard
 type DashboardStatus struct {
 	components.Status `json:",inline"`
+
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Why are we adding namespace here? Wouldn't it be same, `applicationNamespace` for all components?

Reply:
It is a leftover.

I've initially aded it as a POC to include additional information as part of the component status

I'm remove it.

Reply:
done